src/register/components/check-your-answers/check-your-answers.component.html

```diff
@@ -44,8 +44,8 @@ <h1 class="govuk-heading-xl">Check your answers before you register</h1>
          class="govuk-check-your-answers__contents">
       <ng-container question>DX reference</ng-container>
       <ng-container answer>
-        {{ formDataValues.DXnumber }}
-        {{ formDataValues.DXexchange }}
+        DX number: {{ formDataValues.DXnumber }}<br>
+        DX exchange: {{ formDataValues.DXexchange }}
```

Comment:
i really don't like this br tag... that is not very html5... i wonder if we could use span and class from govuk styles to achieve the same